Run chart testing always with image from pr (#82)
kyma-project/helm-broker: charts/helm-broker/ci/README.md

> ## Providing Custom Test Values

Reviewer: ```suggestion
## Providing custom test values
```

In order to enable custom test values, add any number of *-values.yaml files to this directory. Only files with a suffix -values.yaml are considered. Instead of using the defaults, the chart is then installed and tested separately for each of these files using the --values flag.

> Please note that in order to test using the default values when using the ci directory, an empty values file must be present in the directory.

Reviewer: ```suggestion
If you want to perform testing using the default values, an empty `values.yaml` file must be present in the `ci` directory.
```
